nymvpn-x/src-tauri/src/main.rs

```diff
@@ -0,0 +1,197 @@
+// Prevents additional console window on Windows in release, DO NOT REMOVE!!
+#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")]
+
+use std::time::Duration;
+use std::{env, sync::Arc};
+
+use crate::window::WindowSize;
+use crate::{
+    cli::{print_build_info, Cli},
+    db::{Db, Key},
+    fs::{config::AppConfig, storage::AppStorage},
+    grpc::client::GrpcClient,
+    network::setup_network_env,
+};
+
+use anyhow::{anyhow, Result};
+use clap::Parser;
+use commands::daemon as cmd_daemon;
+use commands::db as cmd_db;
+use commands::window as cmd_window;
+use commands::*;
+use states::app::AppState;
+use tauri::{api::path::config_dir, Manager};
+use tokio::sync::Mutex;
+use tokio::time::sleep;
+use tracing::{debug, error, info, trace};
+
+mod cli;
+mod commands;
+mod country;
+mod db;
+mod error;
+mod events;
+mod fs;
+mod gateway;
+mod grpc;
+mod network;
+mod states;
+mod vpn_status;
+mod window;
+
+pub const APP_DIR: &str = "nym-vpn";
```

Comment:
nym-vpn-x, since this doesn't run as root it wont be able to read the filed in that directory if you have used the other client first